src/com/google/enterprise/adaptor/ValidatedUri.java

```diff
@@ -49,8 +49,8 @@
    * @throws URISyntaxException if the URL syntax is invalid
    */
   public ValidatedUri(String uriString) throws URISyntaxException {
-    if (uriString == null) {
-      throw new URISyntaxException("null", "null URI");
+    if (Strings.isNullOrEmpty(uriString)) {
+      throw new URISyntaxException("" + uriString, "null or empty URI");
```

Comment:
This code pattern is like nails on a blackboard for me, but why are you using it *with a string*?